fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -1891,13 +1891,25 @@ void MultiVersionApi::loadEnvironmentVariableNetworkOptions() {
 		return;
 	}
 
+	FDBOptionParamType curParamType;
+	int64_t intParamVal;
```

Comment:
That makes sense, so that the code could look more clean. 
My previous considerations were, if we set them in a higher scope, we don't need to declare it and allocate in memory again and again, and meanwhile we could use the cpu's cache locality to attract the memory efficiently. 
Does it make sense?

Reply:
I think it's generally good practice to declare variables in their lowest scope, and I wouldn't expect there to be a performance concern here unless perhaps you were repeatedly assigning the same value to the variable in the loop.